scripts/install

```diff
@@ -59,6 +59,17 @@ db_name=$(ynh_sanitize_dbid $app)
 ynh_app_setting_set $app db_name $db_name
 ynh_mysql_setup_db $db_name $db_name
 
+# Setup a privileged user for phpmyadmin (to prevent using MySQL root user)
+db_admin_user="${app}_root"
+ynh_app_setting_set $app db_admin_user $db_admin_user
+db_admin_pwd="$(ynh_string_random)"
+ynh_app_setting_set $app db_admin_pwd $db_admin_pwd
+
+if ! ynh_mysql_user_exists "$db_admin_user" ; then
+  ynh_mysql_create_user "$db_admin_user" "$db_admin_pwd"
+  ynh_mysql_execute_as_root "GRANT ALL PRIVILEGES ON *.* TO '$db_admin_user'@'localhost' IDENTIFIED BY '$db_admin_pwd' WITH GRANT OPTION;
+  FLUSH PRIVILEGES;" mysql
```

Comment:
Is it going to be enough to grant all privileges only on "mysql" database ?

Reply:
Actually, in my understanding, the mysql parameter at the very end is needed because user credentials are managed in the mysql database. But the "\*.\*" specifies rights on any databases.

Reply:
Ok
I guess that you have tried it ?

Reply:
Ok perfect then :)